### PR TITLE
Create a Bootstrap Example (#260)

### PR DESCRIPTION
**Pull request by semijacks**
Address #260 

- Similar UI to the with-chakra-ui example but using bootstrap components
- Overrides the default bootstrap colors with the colors defined in mirrorful

image

### CODE REVIEW

File: examples/bootstrap-next/src/pages/_app.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import 'bootstrap/dist/css/bootstrap.min.css'
+import '../../.mirrorful/theme.css'
```
Comment (alexdanilowicz) [inline examples/bootstrap-next/src/pages/_app.tsx:2]:
```suggestion
import '../../.mirrorful/theme.css' // key line that imports the mirrorful theme
```

File: examples/bootstrap-next/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) and includes [Mirrorful](https://www.mirrorful.com/).
```
Comment (alexdanilowicz) [inline examples/bootstrap-next/README.md:1]:
```suggestion
# Bootstrap + Mirrorful + Next

This is a [Next.js](https://nextjs.org/) project from [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) using Boostrap and includes [Mirrorful](https://www.mirrorful.com/).
```

File: examples/bootstrap-next/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+```
+yarn
+```
+
+```bash
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```
Comment (alexdanilowicz) [inline examples/bootstrap-next/README.md:15]: Could you copy the other example READMEs to explain how to run mirrorful? @semijacks

**Review by alexdanilowicz — COMMENTED** [review]
Great work @semijacks. 
 
 Some comments.. I left a code review. Also:
 
 are you also able to add the example:
 
1. here in our docs? https://github.com/Mirrorful/mirrorful/blob/main/docs/home/examples.mdx

2. to the list here, so it shows up in the app? https://github.com/Mirrorful/mirrorful/blob/main/packages/core/src/components/ExportSuccessModal.tsx#L421-L444

**Comment by semijacks** [comment]
> Great work @semijacks.
> 
> Some comments.. I left a code review. Also:
> 
> are you also able to add the example:
> 
> 1. here in our docs? https://github.com/Mirrorful/mirrorful/blob/main/docs/home/examples.mdx
> 2. to the list here, so it shows up in the app? https://github.com/Mirrorful/mirrorful/blob/main/packages/core/src/components/ExportSuccessModal.tsx#L421-L444

File: examples/bootstrap-next/README.md (hunk 3 above)
Comment (semijacks) [reply examples/bootstrap-next/README.md:15]: Alright. I'll do that

**Comment by semijacks** [comment]
Thanks for the feedback @alexdanilowicz. I will add the changes you suggested

**Comment by semijacks** [comment]
Made the changes @alexdanilowicz
Please let me know what you think

**Comment by alexdanilowicz** [comment]
Screen Shot 2023-04-14 at 9 09 37 AM

awesome work, merging this in!

**Pull request merged**